server/controllers/lettersControllers/lettersControllers.js

```diff
@@ -40,9 +45,8 @@ exports.getAllLetters = async (req, res) => {
 
 exports.getLettersById = async (req, res) => {
     try {
-        const letterId = req.params.id
+        const {letterId} = req.params.id
```

Comment:
Looks like we reverted back to a slightly older commit. At least it worked lol! Let's remove the curly braces around letterId again.